odoo_backup_sh/views/odoo_backup_sh_views.xml

```diff
@@ -220,6 +220,9 @@ License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
                     <button name="action_create_simulation_data" string="Create Fake Backups" type="object" class="btn-secondary" attrs="{'invisible': [('backup_simulation', '=', False)]}"/>
                 </header>
                 <sheet>
+                    <label class="text-warning float-right" attrs="{'invisible': [('config_cron_ids', '!=', [])]}">
+                        Backups will not be made until you configure schedule
+                    </label>
```

Comment:
```

  File "/mnt/odoo-extra/runbot/static/build/02724-692-5ed381/odoo/addons/base/models/ir_ui_view.py", line 360, in _check_xml
    raise ValidationError(_('Invalid view %s definition in %s') % (view.name, view.arch_fs))
odoo.tools.convert.ParseError: "Invalid view odoo_backup_sh.config.form definition in odoo_backup_sh/views/odoo_backup_sh_views.xml
```

Reply:
Using `p` instead of `label` https://github.com/it-projects-llc/addons-dev/pull/692/commits/1eca9ebea753d7467249144095db44a0f368a7c3